### ISSUE TITLE
Testing isues of new versions

### ISSUE THREAD

**Issue by Yasen6275** [nextcloud/server#28000]
After updating my old and neglected v19 cloud to v21 and having numerous issues with the upgrade I decided to make clean install and transfer my data 

### Steps to reproduce
1.Install latest version of Debian stable
2.Install all the  necessary apache, php and php modules
3. Configure apache and php

### Expected behaviour
Everything to work without problems.

### Actual behaviour
While waiting to finish the install I was redirected to a page for recommended apps with missing base url . If I open the base url I end up in somewhat working  instance of Nextclod. But in settings/Overview I see more or less the same issues as on my neglected  and upgraded server. The most concerning and  identical on both places beeing webfinger, nodeinfo, caldav and cardav.  
Adding base url to the broken redirec relived me from webfinger and nodeinfo issues.
BUT links to documentation regarding the issues are broken in both cases.

All of the above was tested with and without dedicated folder for nextclod, and when testig with dedicated directory, .htaccess was edited accordingly.

### Server configuration

**Operating system:**
Debian Buster
**Web server:**
Apache
**Database:**
Sqlite
**PHP version:**
7.3
**Nextcloud version:** (see Nextcloud admin page)
22
**Updated from an older Nextcloud/ownCloud or fresh install:**
no
**Where did you install Nextcloud from:**
Is that question politely asking if I;ve downloaded  it from the only appropriate place?
**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
Nothing abnormal there
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
Enabled:
  - accessibility: 1.7.0
  - activity: 2.15.0
  - bruteforcesettings: 2.2.0
  - calendar: 2.3.1
  - circles: 22.0.0
  - cloud_federation_api: 1.4.0
  - comments: 1.11.0
  - contacts: 4.0.0
  - contactsinteraction: 1.2.0
  - cookbook: 0.8.4
  - dashboard: 7.1.0
  - dav: 1.18.0
  - deck: 1.5.0
  - federatedfilesharing: 1.11.0
  - federation: 1.11.0
  - files: 1.16.0
  - files_pdfviewer: 2.3.0
  - files_rightclick: 1.1.0
  - files_sharing: 1.13.2
  - files_trashbin: 1.11.0
  - files_versions: 1.14.0
  - files_videoplayer: 1.11.0
  - firstrunwizard: 2.11.0
  - logreader: 2.7.0
  - lookup_server_connector: 1.9.0
  - mail: 1.10.1
  - nextcloud_announcements: 1.11.0
  - notifications: 2.10.1
  - oauth2: 1.9.0
  - password_policy: 1.12.0
  - photos: 1.4.0
  - privacy: 1.6.0
  - provisioning_api: 1.11.0
  - recommendations: 1.1.0
  - richdocuments: 4.2.0
  - serverinfo: 1.12.0
  - settings: 1.3.0
  - sharebymail: 1.11.0
  - spreed: 12.0.1
  - support: 1.5.0
  - survey_client: 1.10.0
  - systemtags: 1.11.0
  - text: 3.3.0
  - theming: 1.12.0
  - twofactor_backupcodes: 1.10.1
  - updatenotification: 1.11.0
  - user_status: 1.1.1
  - viewer: 1.6.0
  - weather_status: 1.1.0
  - workflowengine: 2.3.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_ldap

</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "bananapim2zero.lan"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "sqlite3",
        "version": "22.0.0.11",
        "overwrite.cli.url": "http:\/\/bananapim2zero.lan",
        "installed": true,
        "app_install_overwrite": [
            "cookbook"
        ]
    }
}

or 

</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
nope
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no
#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';

Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**
Firefox
**Operating system:**
Fedora 33 mate

In conclusion I have two questions.

Have someone from your team tested version 22 on bare metal before release?
Are you sure 22 is stable version and not still in beta/rc/whatever?

**Comment by Yasen6275** [comment]
Just realized what is " Where did you install Nextcloud from:"
From zip file.

**Comment by kesselb** [comment]
Closing in favor of #27575
